backend/database_handler/transactions_processor.py

```diff
@@ -149,7 +149,6 @@ def set_transaction_result(self, transaction_hash: str, consensus_data: dict):
             self.session.query(Transactions).filter_by(hash=transaction_hash).one()
         )
 
-        transaction.status = TransactionStatus.FINALIZED
```

Comment:
What's the reason behind this change?

Reply:
It was redundant because we also update the status in the update status method, and I thought it was causing the unit test to fail. But actually updating the mocking in the test was enough, so I'm reverting this! nevermind